src/Graphics/samples/GraphicsTester.WinUI.Desktop/GraphicsTester.WinUI.Desktop.csproj

```diff
@@ -20,7 +20,7 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <WindowsSdkPackageVersion>10.0.19041.44</WindowsSdkPackageVersion>   
+    <WindowsSdkPackageVersion>10.0.19041.56</WindowsSdkPackageVersion>   
```

Comment:
Why it can t be 10.0.19041.57 ?

Reply:
We can make that them 57 too, it probably was different versions before due to not having the SDK profile changes.  But for testing purposes here, 56 or 57 here are fine.